letsencrypt/rootfs/etc/cont-init.d/file-structure.sh

```diff
@@ -80,3 +80,17 @@ fi
 if bashio::config.exists 'dns.cloudxns_secret_key'; then
       bashio::addon.option 'dns.cloudxns_secret_key'
 fi
+
+# ClouDNS
+# Only a single non-empty auth option must be in /data/dnsapikey.
+# User should pick just one (as per examples). If they pick more than one by accident, they will get an error
```

Comment:
I think this is already in the documentation. If you want to do something more elaborate, it would be better to proactively in https://github.com/home-assistant/addons/blob/668e6fad5d1eae56d009c733bfc0d7281a8446d1/letsencrypt/rootfs/etc/services.d/lets-encrypt/run#L167-L168. E.g. we can use `bashio::config.has_value`.

Reply:
ClouDNS is properly documented but currently all three "auth type" keys are left in /data/dnsapikey (values empty by default). Certbot halts if more than one key for ClouDNS exist /data/dnsapikey regardless if only has has a value. Note that this only happens if you use any of the ClouDNS auth types. 

I'm unsure what you meant by "more elaborate" as this is a bug fix for the current implementation (duct-tape as it may be). I suppose I could move the logic to the `run` file. It's my first time contributing so I don't yet have a good feel of the project.

Reply:
If you look a few lines above my added code, there is already some post-processing to the `/data/dnsapikey` being done here, which is why I felt this is the appropriate place to do so. I would appreciate some guidance to as why it would be more suitable for this logic to be moved to the `run` file